### DISCUSSION TITLE
ASCII Terminal Banner

### DISCUSSION THREAD

**Discussion by pstreich** [quasarframework/quasar#16238] [General - Components / Directives / etc] [answered]
Hi,

Is there a way to customise the 'Quasar' ASCII Banner which is shown in the terminal on app launch?

**Comment by yusufkandemir** [comment, marked as answer]
Not unless you fork it and use your own version or use something like `pnpm patch`.

It's being printed here: https://github.com/quasarframework/quasar/blob/a4bc2e42e1b5c52080913fe6d678339d20949b13/app-webpack/lib/cmd/dev.js#L84-L89
And here is the ASCII art:
https://github.com/quasarframework/quasar/blob/a4bc2e42e1b5c52080913fe6d678339d20949b13/app-webpack/assets/logo.art

Can you share why you want to customize it?

**Reply by pstreich** [reply]
Thanks for the blazing fast reply :)
There is no specific, functional reason.
When doing spring boot projects, I found that the team always liked a customised banner.
Just a gimmick to boost identity formation, I guess.

**Reply by rstoenescu** [reply]
I'd recommend making a script to run (npm/yarn/etc) post-install to replace the logo.art file.
Stay up to date with q/app-vite or q/app-webpack otherwise you'll miss important fixes and new features.
